### PR TITLE
Simplifying the Bool creation with NSNumber

### PR DESCRIPTION
**Pull request by frootloops**

### CODE REVIEW

File: test/1_stdlib/Bool.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
```
Comment (gribozavr) [inline test/1_stdlib/Bool.swift]: Also `// REQUIRES: objc_interop`.

File: test/1_stdlib/Bool.swift [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// RUN: %target-run-simple-swift
```
Comment (gribozavr) [inline test/1_stdlib/Bool.swift]: It would be better to call the test "BoolBridge.swift".

**Comment by frootloops** [comment]
@gribozavr done

**Comment by gribozavr** [comment]
@frootloops Thank you!

**Pull request merged**